snapshot-labs/snapshot
# feat: Remove mixpanel

### Summary



Closes: https://github.com/snapshot-labs/workflow/issues/188

### How to test

1. Go to the home page, and notice the console network tab, it should not send any tracking messages like before
2. Also an many actions, for example while voting
